### PR TITLE
Updated Node-Menu

### PR DESCRIPTION
**Pull request by 3DSinghVFX**
This branch has updated node_menu.py file. It has all the nodes in the "Add" node-menu.

### CODE REVIEW

**Comment by OmarEmaraDev** [comment]
We don't want to increase the length of the add menu too much. Perhaps we should put the collections menu in the Object menu, in place of the Collection Info node.

Also, the *Interface Menu* doesn't really make sense here. The two nodes inside are different types of interfaces. Perhaps they should go to the subprograms menu.

**Comment by 3DSinghVFX** [comment]
Okay. I have updated the "Add" node-menu.

File: animation_nodes/ui/node_menu.py [hunk 1]
```diff
@@ -201,6 +201,7 @@ def draw(self, context):
         layout.separator()
         insertNode(layout, "an_TextBlockReaderNode", "Block Reader")
         insertNode(layout, "an_TextBlockWriterNode", "Block Writer")
+        insertNode(layout, "an_FilterBlendDataListByNameNode", "Filter Text Block List", {"dataType" : repr("Text")})
```
Comment (OmarEmaraDev) [inline animation_nodes/ui/node_menu.py]: Shouldn't this be `Text Block` instead of `Text`?

Comment (3DSinghVFX) [reply animation_nodes/ui/node_menu.py]: I also thought that but in the filter node, the data type is written as "Text Block".

Comment (OmarEmaraDev) [reply animation_nodes/ui/node_menu.py]: I mean it should be `repr("Text Block")`. `Text` is not supported by the node as far as I know.

Comment (3DSinghVFX) [reply animation_nodes/ui/node_menu.py]: Sorry, I misunderstood. Yes, it is `Text Block`. However, it is also working with `Text`.

Comment (3DSinghVFX) [reply animation_nodes/ui/node_menu.py]: It is fixed.

Comment (OmarEmaraDev) [reply animation_nodes/ui/node_menu.py]: Good. But I don't think it works with `Text`, str doesn't have a name attribute.

Comment (3DSinghVFX) [reply animation_nodes/ui/node_menu.py]: The node is not working properly. But it does not show any error,
Screenshot from 2019-12-29 17-51-12

**Pull request merged**